### PR TITLE
DDP 4857 covid survey updates

### PR DESCRIPTION
**Pull request by ssettipalli**
Add new question to testboston covid-survey as in https://broadinstitute.atlassian.net/browse/DDP-4857

### CODE REVIEW

File: study-builder/studies/testboston/subs.conf [hunk 1]
```diff
@@ -26,6 +26,15 @@
       "self_sig": "SELF_SIGNATURE",
       "staff_sig": "STAFF_SIGNATURE",
       "interpreter_sig": "INTERPRETER_SIGNATURE",
+      "covid_dob": "COVID_SURVEY_DOB",
```
Comment (yufengwng) [inline study-builder/studies/testboston/subs.conf]: Can this just be `DOB` or `BIRTH_DATE`? We don't ask this question anywhere else, so it should be unique enough?

File: study-builder/studies/testboston/i18n/en.conf [hunk 2]
```diff
@@ -87,6 +87,17 @@
   "covid_test_kind_prompt": "What kind of test(s) did you get? (Check all that apply)",
   "covid_test_date_prompt": "When did you get tested?",
   "covid_test_result_prompt": "What was the result of your COVID-19 test?",
+  "covid_dob_prompt": "What is your date of birth?",
+  "sex_prompt": "What sex were you assigned at birth?",
+  "gender_prompt": "What is your gender? ",
+  "gender_tip": "Some people have gender identities that differ from their sex assigned at birth. What is your gender identity?"
+  "ethnicity_prompt": "What is your ethnicity?",
+  "race_prompt": "What is your race (select all that apply)??",
```
Comment (yufengwng) [inline study-builder/studies/testboston/i18n/en.conf]:
```suggestion
  "race_prompt": "What is your race (select all that apply)?",
```

File: study-builder/studies/snippets/picklist-question-yndk-with-translations.conf [hunk 3]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (yufengwng) [inline study-builder/studies/snippets/picklist-question-yndk-with-translations.conf]: I don't see this snippet being used?

File: study-builder/studies/testboston/snippets/question-household-member-age.conf [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          { "language": "es", "text": ${i18n.es.household_member_add} },
+          { "language": "ht", "text": ${i18n.ht.household_member_add} }
+        ]
+      }
+    ]
+  },
+  "children": [
+    ${_includes.covid_survey.household_member},
+    ${_includes.covid_survey.age},
+  ]
```
Comment (yufengwng) [inline study-builder/studies/testboston/snippets/question-household-member-age.conf]: I got a bit confused about the naming. Maybe we should have:
```
question-household-list.conf
  - question-household-member.conf
  - question-household-member-age.conf
```

**Review by yufengwng — COMMENTED** [review]
A few small comments.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**